src/models/Expense.model.js

```diff
@@ -1,9 +1,45 @@
 'use strict';
 
+const { DataTypes } = require('sequelize');
 const { sequelize } = require('../db.js');
 
 const Expense = sequelize.define(
-  // your code goes here
+  'Expense',
+  {
+    id: {
+      type: DataTypes.INTEGER,
+      primaryKey: true,
+      autoIncrement: true,
+    },
+    userId: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+    },
+    spentAt: {
+      type: DataTypes.DATE,
+      defaultValue: DataTypes.NOW,
+      allowNull: false,
+    },
+    title: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    amount: {
+      type: DataTypes.FLOAT,
+      allowNull: false,
+    },
+    category: {
+      type: DataTypes.STRING,
+    },
+    note: {
+      type: DataTypes.STRING,
+    },
+  },
+  {
+    tableName: 'expenses',
+    updatedAt: false,
+    createdAt: false,
```

Comment:
Consider enabling `createdAt` and `updatedAt` timestamps to automatically track when records are created and modified. This can be useful for auditing and historical data analysis.